docs/b2b-edition/specs/api-v3/company/company.yaml

```diff
@@ -2326,12 +2290,12 @@ paths:
                   description: Get addresses
                   code: get_addresses
                   moduleName: address
-        description: 'name, code are unique in the same store.'
+        description: "name, code are unique in the same store."
```

Comment:
```suggestion
        description: "name, code is unique in the same store."
```

Reply:
I actually think it should be:

The name and code are unique within the same store.